manifests/010-prometheus-rules.yaml

```
@@ -40,7 +74,10 @@ spec:
         severity: warning
       annotations:
         message: |
-          Samples operator cannot find credentials for 'registry.redhat.io' in the samples pull secret in the openshift namespace.  Unless you have updated the 'samplesRegistry' field in the samples operator config to change which registry the sample ImageStreams are pulled from, you will have failed imports for many of the sample ImageStreams in this case.
+          Samples operator cannot find credentials for 'registry.redhat.io' in the samples pull secret in the openshift
+          namespace.  Unless you have updated the 'samplesRegistry' field in the samples operator config to change which
+          registry the sample ImageStreams are pulled from, you will have failed imports for many of the sample ImageStreams
+          in this case.
```

I likewise think we make this more concise (even though this was an existing alert):
```suggestion
          The samples operator cannot find credentials for 'registry.redhat.io'. Many of the sample ImageStreams will fail to import unless the 'samplesRegistry' in the operator configuration is changed.
```